### PR TITLE
Modifed 'Configuring DNS options to provide instructions for Ubuntu

### PR DESCRIPTION
**Pull request by zelahi**
Modified ubuntulinux.md for instructions to configure DNS for Ubuntu 15.04 or greater. For Ubuntu 15.04 the user needs to use a drop-in file to configure the DNS Server.  I provided instructions on where to place the drop-in file for this.

@moxiegirl and @thaJeztah as of now I added it as a note, but I was thinking that if you guys wanted I could change step 2 to use the drop-in file instead of modifying the file at /etc/default/docker

Let me know what you guys think.  I also provided the minimum amount of information to get this to work for the users so if I missed any steps I am happy to add them. Finally let me know if I am missing any information

Ref Issue #17586 

Signed-off-by: Zuhayr Elahi elahi.zuhayr@gmail.com

### CODE REVIEW

File: docs/installation/ubuntulinux.md [hunk 1]
```diff
@@ -357,6 +357,8 @@ To specify a DNS server for use by Docker:
 
         $ sudo nano /etc/default/docker
 
+>**Note**:For  Ubuntu 15.04, place a drop-in file in   `/etc/systemd/system/docker.service.d` and modify that with the properties below instead of working on the `/etc/default/docker` file.
```
Comment (thaJeztah) [inline docs/installation/ubuntulinux.md]: Maybe we can prevent using a "note", by adding something before line 352, e.g.:

```Markdown
The instructions below describe how to configure a daemon running on
Ubuntu 14.10 or below. Ubuntu 15.04 and above uses systemd as its boot
and service manager. Refer to [control and configure Docker with systemd](../articles/systemd.md#custom-docker-daemon-options)
to configure a daemon controled by systemd.
```

wdyt?

**Comment by thaJeztah** [comment]
Thanks @zelahi ! I added a suggestion above, but perhaps @moxiegirl has a better suggestion.

I think that eventually we should move the "Adjust memory and swap accounting" and "Configure a DNS server for use by Docker" to another location, because they're general information, and can be used on other distributions as well.

**Comment by zelahi** [comment]
okay, 

if no one else has suggestions I can go ahead and change his based on your suggestions

**Comment by thaJeztah** [comment]
@zelahi most people may not be online because of thanksgiving/black friday :smile:

**Comment by zelahi** [comment]
I figured so no worries. I can wait =)

File: docs/installation/ubuntulinux.md (hunk 1 above)
Comment (moxiegirl) [inline docs/installation/ubuntulinux.md]:
```Markdown
The instructions below describe how to configure the Docker daemon running on
Ubuntu 14.10 or below. Ubuntu 15.04 and above use `systemd` as the boot
and service manager. Refer to [control and configure Docker with systemd](../articles/systemd.md#custom-docker-daemon-options)
to configure a daemon controlled by `systemd`.
```

**Comment by zelahi** [comment]
okay I made the modification as requested.  Hope all looks good =)

File: docs/installation/ubuntulinux.md [hunk 2]
```diff
@@ -349,6 +349,12 @@ To avoid this warning, you can specify a DNS server for use by Docker
 containers. Or, you can disable `dnsmasq` in NetworkManager. Though, disabling
 `dnsmasq` might make DNS resolution slower on some networks.
 
+
+The instructions below describe how to configure a daemon running on
+Ubuntu 14.10 or below. Ubuntu 15.04 and above uses systemd as its boot
+and service manager. Refer to [control and configure Docker with systemd](../articles/systemd.md#custom-docker-daemon-options)
+to configure a daemon controled by systemd.
```
Comment (thaJeztah) [inline docs/installation/ubuntulinux.md]: Looks like you missed some changes (e.g. "a daemon" -> "the Docker daemon"). I also see a typo; controled -> controlled.

Can you copy/paste @moxiegirl's example?

**Comment by thaJeztah** [comment]
Thanks @zelahi, I left another comment. Also, can you "squash" your commits, so that's there just a single commit in this PR? You can find some instructions in our [contribution guide](https://docs.docker.com/opensource/workflow/work-issue/#pull-and-rebase-frequently)

**Comment by zelahi** [comment]
okay done, document has been updated and commits squashed =)

**Comment by moxiegirl** [comment]
@zelahi I think systemd needs tics but we can carry that for you.  @thaJeztah if you are cool with that carry LGTM on my end.

**Comment by thaJeztah** [comment]
carried in https://github.com/docker/docker/pull/18367, I'll close this PR

**Pull request closed without merging**